CRM/Financial/BAO/Order.php

```diff
@@ -742,22 +760,45 @@ public function getLineItem($index): array {
    */
   protected function fillMembershipLine(array $lineItem): array {
     $fields = $this->getPriceFieldsMetadata();
-    $field = reset($fields);
```

Comment:
this incorrectly assumed there would only be one possible field - but it's one per membership organization (the field in the membership_type table)